include/worker.hpp

```diff
@@ -100,6 +100,16 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
+     * @brief API to get VPD config JSON object
+     *
+     * @return parsed VPD config JSON object
```

Comment:
> VPD

Not a VPD config JSON, it is system config JSON.

Reply:
done